lib/ProductOpener/Ingredients.pm

```diff
@@ -3580,7 +3580,9 @@ reference to the name of the country
 sub get_geographical_area ($originid) {
 	# Getting information about the country
 	my $ecobalyse_area = "";
-	if (get_inherited_property("countries", $originid, "ecobalyse_is_part_of_eu") eq "yes") {
+	if (defined get_inherited_property("countries", $originid, "ecobalyse_is_part_of_eu")
```

Comment:
Can you call get_inherited_property("countries", $originid, "ecobalyse_is_part_of_eu") only once?

my $ecobalyse_is_part_of_eu = get_inherited_property("countries", $originid, "ecobalyse_is_part_of_eu");

then check it is defined and equal to "yes"